File: code-coaches/easy/popsicles/popsicles.rb
def distribute_popsicles(popsicles, siblings)
  if (popsicles % siblings == 0)
    return "give away"
  else
    return "eat them yourself"
  end
end

def main
  siblings = 0

  until siblings != 0
    siblings = gets.chomp.to_i
  end

  popsicles = gets.chomp.to_i

  puts distribute_popsicles(popsicles, siblings)
end

if __FILE__ == $0
  main
end

File: code-coaches/easy/popsicles/popsicles.c
#include <stdio.h>

const char* distribute_popsicles(int popsicles, int siblings)
{
    if (popsicles % siblings == 0)
		return "give away";
	else
		return "eat them yourself";
}

int main()
{
    int siblings, popsicles;

	do
	{
        scanf("%d %d", &siblings, &popsicles);
	}
	while (siblings == 0);

	printf("%s", distribute_popsicles(popsicles, siblings))

	return 0;
}
